ses-ballerina/src/ses/ses.bal

```diff
@@ -0,0 +1,599 @@
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/email;
+import ballerina/encoding;
+import ballerina/http;
+import ballerina/lang.array;
+import ballerina/time;
+
+# Initializes the AWS SES client based on the provided configurations.
+public type Client client object {
+
+    private string accessKey;
+    private string secretKey;
+    private string region;
+    private string host;
+    private http:Client clientEp;
+    private email:SmtpClient? smtpClient = ();
+
+    # Initializes the AWS SES client based on the provided configurations.
+    #
+    # + config - The configurations for the AWS SES client
+    public function init(Configuration config) {
+        self.accessKey = config.accessKey;
+        self.secretKey = config.secretKey;
+        self.region = config.region;
+        self.host = string `${SES_SERVICE_NAME}.${self.region}.${AMAZON_HOST}`;
+        http:ClientSecureSocket? clientSecureSocket = config?.secureSocketConfig;
+        if (clientSecureSocket is http:ClientSecureSocket) {
+            self.clientEp = new(HTTPS_URL_PREFIX + self.host, {secureSocket: clientSecureSocket});
+        } else {
+            self.clientEp = new(HTTPS_URL_PREFIX + self.host, {});
+        }
+
+        string smtpPassword = self.getSmtpPassword();
+        string smtpUsername = self.accessKey;
+        int smtpPort = 587;
+        string smtpPortString = smtpPort.toString();
+        email:SmtpConfig smtpConfig = {
+            port: smtpPort,
+            enableSsl: false,
+            properties: {
+                PROP_MAIL_TRANSPORT_PROTOCOL:SMTP,
+                PROP_MAIL_SMTP_PORT:smtpPortString,
+                PROP_MAIL_SMTP_STARTTLS_ENABLE:"true", PROP_MAIL_SMTP_AUTH:"true"
+            }
+        };
+        string awsSmtpHost = string `${SES_SMTP_SERVICE_NAME}.${self.region}.${AMAZON_HOST}`;
+        self.smtpClient = new (awsSmtpHost, smtpUsername, smtpPassword, smtpConfig);
+
+    }
+
+    # Verifies the given email address by sending a verification email to it.
+    # ```ballerina
+    # ses:Error? result = sesClient->verifyEmailIdentity("a@bcd.com");
+    # ```
+    #
+    # + emailAddress - The email address to be verified
+    # + return - A `ses:Error` if an error occurred while the operation
+    public remote function verifyEmailIdentity(string emailAddress) returns
+            Error? {
+        string endpoint = DEFAULT_ENDPOINT;
```

Comment:
IMO, we don't need to define a new variable. we can directly use the constant

Reply:
Changed.